cmd/webhook/builder.go

```diff
@@ -179,7 +179,7 @@ func (builder CommandBuilder) buildRun() func(*cobra.Command, []string) error {
 			return err
 		}
 
-		err = edgeconnectvalidationhook.AddEdgeConnectValidationWebhookToManager(webhookManager)
+		err = edgeconnectv1alpha1validation.AddEdgeConnectValidationWebhookToManager(webhookManager)
```

Comment:
should me same as for new dynakube:
```
package dynakube

import (
	ctrl "sigs.k8s.io/controller-runtime"
	"sigs.k8s.io/controller-runtime/pkg/webhook/admission"
)

func SetupWebhookWithManager(mgr ctrl.Manager, validator admission.CustomValidator) error {
	return ctrl.NewWebhookManagedBy(mgr).
		For(&EdgeConnect{}).
		WithValidator(validator). // will create an endpoint at /validate-edgeconnect-com-v1beta1-edgeconnect
		Complete()
}

```

Reply:
fixed in 6788c74cf9a36ee7b29223708f082dc163869b3e